reset-dev-env.bash

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+set -o errexit
+
+cd "$(dirname "${BASH_SOURCE[0]}")"
+
+npm install
+
+poetry install --extras='cdk datasets-endpoint'
+
+# shellcheck source=/dev/null
+. .venv/bin/activate
+
+pre-commit install --hook-type=commit-msg --overwrite
+pre-commit install --hook-type=pre-commit --overwrite
```

Comment:
It would be great if this script would add `cdk` executable to  `PATH`

Reply:
That won't propagate to the parent shell. But we should include instructions for doing that.

Reply:
OK, thanks

Reply:
Script tested and works.